sdk/js/generated/allowed-field-masks.json

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
Do we actually need this file to be added to Git?

Reply:
Since it is rarely edited, I believe we would be saving some build time by embedding it. But it can be removed and be made part of the JS SDK build pipeline.